app/src/main/java/chat/rocket/android/main/ui/Menu.kt

```diff
@@ -0,0 +1,78 @@
+package chat.rocket.android.main.ui
+
+import android.view.Menu
+import android.view.MenuItem
+import chat.rocket.android.R
+
+private const val MENU_SECTION_ONE = 1
+private const val MENU_SECTION_TWO = 2
+private const val MENU_SECTION_THREE = 3
+
+const val MENU_ACTION_CHATS = 1
+private const val MENU_ACTION_CREATE_CHANNEL = 2
+const val MENU_ACTION_PROFILE = 3
+private const val MENU_ACTION_SETTINGS = 4
+private const val MENU_ACTION_ADMIN_PANEL = 5
+private const val MENU_ACTION_LOGOUT = 6
+
+internal fun MainActivity.setupMenu(menu: Menu) {
+    menu.add(
```

Comment:
`with(menu)` would save a lot of `menu.` 😀

Reply:
Good usage of with as discussed internally.
Changed!
e190306